### PR TITLE
Add a function to compute the spatial intersection of a segment with a VoxelData object

### PR DESCRIPTION
**Pull request by adrien-berchet**

### CODE REVIEW

**Comment by adrien-berchet** [comment]
Hey there,
Is it possible to merge this and release a new version please?

File: voxcell/math_utils.py [hunk 1]
```diff
@@ -165,3 +166,148 @@ def mat2euler(mm):
...
+        seg_pt_end = (seg_points == end_pt).all(axis=1)
+        if seg_pt_end.any():
+            seg_points = seg_points[~seg_pt_end]
+
+    if all([x_ascending, y_ascending, z_ascending]):
+        seg_pt_start = (seg_points == start_pt).all(axis=1)
+        if seg_pt_start.any():
+            seg_points = seg_points[~seg_pt_start]
+
+    # Build and sort the sub-segment points
```
[the remark this answers is not in the document]
Comment (mgeplf) [reply voxcell/math_utils.py:278]: let's add that as a comment

**Comment by mgeplf** [comment]
Sorry, this fell off my radar.

I want to go through the math a bit better, but should happen soon.

**Comment by adrien-berchet** [comment]
Ok, no problem :)

I pushed a new commit to add the requested comment (https://github.com/adrien-berchet/voxcell/commit/21dd411922b6a262ac010323ad583faea4e80303) but it does not appear here. Do you know why? :thinking:

**Comment by mgeplf** [comment]
> I pushed a new commit to add the requested comment ([adrien-berchet/voxcell@21dd411](https://github.com/adrien-berchet/voxcell/commit/21dd411922b6a262ac010323ad583faea4e80303)) but it does not appear here. Do you know why?

Not sure; also, I can't make changes to the PR, so it would be helpful to allow maintainers to change it, if you can check that box.

**Comment by adrien-berchet** [comment]
Maybe it's because this PR was opened when the repository was private and the fork sync was kinda lost when it was made public? I made my fork public too but it didn't fix this issue.

**Comment by mgeplf** [comment]
> The box is already checked

Strange, because I can push to the branch.  Was this PR made before the repo was public?  I don't know.

I added you with write access to the repo; if you want to push the branch here, that will probably be easier.

**Comment by adrien-berchet** [comment]
Ok, I opened a new PR with the new commit: #11

**Pull request closed without merging**